### PR TITLE
Mapping documents -> zaakinformatieobjecten

### PR DESCRIPTION
**Pull request by bbrands02**

### CODE REVIEW

File: src/Service/ZaakService.php [hunk 1]
```diff
@@ -191,6 +191,59 @@ private function checkZaakType(array $case)
     }//end checkZaakType()
 
 
+    /**
+     * Gets the actual document from a different endpoint that has the metadata.
+     *
+     * @param string $documentNumber document number (not the id).
+     *
+     * @return array
```
Comment (WilcoLouwerse) [inline src/Service/ZaakService.php:199]: You could add docs for return value

File: src/Service/ZaakService.php [hunk 2]
```diff
@@ -191,6 +191,59 @@ private function checkZaakType(array $case)
...
+     */
+    private function getActualDocument(string $documentNumber): array
+    {
+        try {
+            isset($this->style) === true && $this->style->info("Fetching actual document: $documentNumber..");
```
Comment (WilcoLouwerse) [inline src/Service/ZaakService.php:204]: (Do we ever reach this code using a command? If not remove $this->style)

**Review by rjzondervan — COMMENTED** [review]
Just one question: I don't really see you adding the document content to the 'enkelvoudigInformatieObject', is that correct, or am I overlooking it?

**Review by WilcoLouwerse — APPROVED** [review]
Ziet er goed uit 👍

File: src/Service/ZaakService.php (hunk 2 above)
Comment (bbrands02) [reply src/Service/ZaakService.php:204]: we do :)

**Comment by bbrands02** [comment]
> Just one question: I don't really see you adding the document content to the 'enkelvoudigInformatieObject', is that correct, or am I overlooking it?

That's not happening yet. I've send a question to xxllnc whether they can come up with a case where documents are attached with content / data because I didn't have that right now.

**Review by rjzondervan — APPROVED** [review]

File: Installation/Source/xxllnc.zaaksysteemv2.source.json [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{
+    "title": "xxllnc zaaksysteem v2",
+    "$id": "https://development.zaaksysteem.nl/source/xxllnc.zaaksysteemv2.source.json",
+    "$schema": "https://docs.commongateway.nl/schemas/Gateway.schema.json",
+    "version": "0.0.1",
+    "description": "xxllnc zaaksysteem v2 api",
+    "location": "!ChangMe!",
+    "authorizationHeader": "API-KEY",
```
Comment (WilcoLouwerse) [inline Installation/Source/xxllnc.zaaksysteemv2.source.json:8]: The correct info for this Source is present in Passwork right?

File: composer.json [hunk 4]
```diff
@@ -16,7 +16,7 @@
     "minimum-stability" : "dev",
     "require": {
         "php": ">=7.4",
-        "commongateway/corebundle": ">=1.0.107",
+        "commongateway/corebundle": ">=1.1.19",
```
Comment (WilcoLouwerse) [inline composer.json:19]: Did you test this with the latest versions of the CoreBundle?
Or is xxllnc Gateway using a specific CoreBundle version 1.1.19 and not higher?

**Review by WilcoLouwerse — APPROVED** [review]
Just 2 questions, looks fine

File: Installation/Source/xxllnc.zaaksysteemv2.source.json (hunk 3 above)
Comment (bbrands02) [reply Installation/Source/xxllnc.zaaksysteemv2.source.json:8]: correct

**Pull request merged**